From febb20e23e5a1152eea47640ccadbca5d34b5218 Mon Sep 17 00:00:00 2001
From: Simon Bray <sbray1371@gmail.com>
Date: Wed, 12 Feb 2020 09:03:15 +0100
Subject: [PATCH] patch

---
 scripts/acpype.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/scripts/acpype.py b/scripts/acpype.py
index a12a0c2..b7fc8f4 100644
--- a/scripts/acpype.py
+++ b/scripts/acpype.py
@@ -94,7 +94,7 @@ if sys.version_info < (3, 6):
     sys.exit(1)
 
 year = datetime.today().year
-__updated__ = "2019-10-08T09:09:22CEST"
+__updated__ = "2019-11-07T23:16:00CET"
 # tag = "2019-09-26T19:44:00UTC"
 tag = __updated__
 
@@ -861,7 +861,7 @@ class AbstractTopol():
             self.printWarn("no charge value given, trying to guess one...")
             mol2FileForGuessCharge = self.inputFile
             if self.ext == ".pdb":
-                cmd = '%s -ipdb %s -omol2 %s.mol2' % (self.babelExe, self.inputFile, self.baseName)
+                cmd = '%s -ipdb %s -omol2 -O %s.mol2' % (self.babelExe, self.inputFile, self.baseName)
                 self.printDebug("guessCharge: " + cmd)
                 out = _getoutput(cmd)
                 self.printDebug(out)
@@ -1463,7 +1463,7 @@ Usage: antechamber -i   input file name
         '''Execute babel'''
         self.makeDir()
 
-        cmd = '%s -ipdb %s -omol2 %s.mol2' % (self.babelExe, self.inputFile,
+        cmd = '%s -ipdb %s -omol2 -O %s.mol2' % (self.babelExe, self.inputFile,
                                               self.baseName)
         self.printDebug(cmd)
         self.babelLog = _getoutput(cmd)
@@ -3426,7 +3426,7 @@ class ACTopol(AbstractTopol):
         self.tleapExe = which('tleap') or ''
         self.sleapExe = which('sleap') or ''
         self.parmchkExe = which('parmchk2') or ''
-        self.babelExe = which('babel') or ''
+        self.babelExe = which('obabel') or which('babel') or ''
         if not os.path.exists(self.babelExe):
             if self.ext != '.mol2' and self.ext != '.mdl':  # and self.ext != '.mol':
                 self.printError("no 'babel' executable; you need it if input is PDB")
-- 
2.17.1

